jest.config.ts

```diff
@@ -1,6 +1,17 @@
 const config = {
+  collectCoverage: true,
   preset: 'ts-jest',
   testEnvironment: 'jsdom',
+  collectCoverageFrom: ['**/*.{ts,js}', '!coverage/**/*.{ts,tsx,js,jsx}'],
+  coverageThreshold: {
+    './src/': {
+      statements: 0,
```

Comment:
Why are these thresholds kept as 0 ?

Reply:
unit test PRs are still not merged to master. After merging UTs of currency and phone number, we can incrementally increase the threshold.